### PR TITLE
Add architecture description and plot

### PR DESCRIPTION
**Pull request by PhilippeMoussalli**
#

### CODE REVIEW

File: docs/architecture.md [hunk 1]
```diff
@@ -1,10 +1,48 @@
 # Architecture
 
-An overview of the architecture of Fondant 
+### Fondant architecture overview
 
-### Coming soon 
+<p align="center">
+    <img src="https://raw.githubusercontent.com/ml6team/fondant/main/docs/art/architecture.png"/>
```
Comment (RobbeSneyders) [inline docs/architecture.md:6]: We can use relative links everywhere but the README.

**Review by RobbeSneyders — COMMENTED** [review]
Thanks @PhilippeMoussalli!

Can you add a link to the architecture docs in the "Contributing" part of the README?

**Comment by PhilippeMoussalli** [comment]
> Thanks @PhilippeMoussalli!
> 
> Can you add a link to the architecture docs in the "Contributing" part of the README?

Done :), had to modify the plot to make the font bigger because it appeared small in the docs

**Review by RobbeSneyders — APPROVED** [review]
Thanks! The failure in the pipeline seems unrelated.

**Pull request merged**